2000_Furusawa/data/output/wdir/12C_V79_117/geo.dat

```diff
@@ -0,0 +1,32 @@
+*---><---><--------><------------------------------------------------>
```

Comment:
...also you shoud not commit generated files.

Reply:
I agree